Makefile

```diff
@@ -14,6 +14,7 @@ ci-build-misc: \
 	clean-tools \
 	proto \
 	go-generate \
+	buf-breaking \
```

Comment:
Can we put a comment here that says to not remove this, and add ignores to `proto/internal/buf.yaml` instead?